agent/ebs/watcher_test.go

```diff
@@ -300,6 +304,104 @@ func TestHandleDuplicateEBSAttachment(t *testing.T) {
 	assert.True(t, ebsAttachment.IsAttached())
 }
 
+func TestStageAll(t *testing.T) {
+	tcs := []struct {
+		name                     string
+		setTaskStateExpectations func(*mock_dockerstate.MockTaskEngineState)
+		setCSIClientExpectations func(*mock_csiclient.MockCSIClient)
+		expectedErrorStubs       []string
+		expectedAttachmentStatus attachment.AttachmentStatus
+	}{
+		{
+			name: "StageAll succeeds with expected EBS CSI Driver fields",
```

Comment:
note: I'll add a case with an error in staging followed by success so we can test that we process past errors in our array.